@piranna I'm unable to install `eventtarget`: 

```
npm ERR! Error: shasum check failed for /home/ack/tmp/npm-14413-3rQ7A2t_/1393459358841-0.8358576616737992/tmp.tgz
npm ERR! Expected: ee88e458920ce9217d2c69c2931580a2b67f2962
npm ERR! Actual:   12c422babde122bf240c2afac662b53279468529
npm ERR! From:     https://registry.npmjs.org/eventtarget/-/eventtarget-0.0.0.tgz
npm ERR!     at /usr/lib/node_modules/npm/node_modules/sha/index.js:38:8
npm ERR!     at ReadStream.<anonymous> (/usr/lib/node_modules/npm/node_modules/sha/index.js:85:7)
npm ERR!     at ReadStream.EventEmitter.emit (events.js:117:20)
npm ERR!     at _stream_readable.js:920:16
npm ERR!     at process._tickCallback (node.js:415:13)
```

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "bindings": "~1.1.1",
-    "eventtarget": "0.0.0",
+    "eventtarget": "~0.0.1",
     "nan": "~0.4.4",
     "node-gyp": "~0.10.10",
     "rsvp": "~3.0.3"

